.gitignore

```diff
@@ -0,0 +1,3 @@
+# Xojo (formerly REALbasic and Real Studio)
+
+*.md
```

Comment:
- [ ] [ OPTIONAL] According to the project requirement you should make sure that  `test.md`  and `node_modules`  are ignored by git and is not present in your GitHub repository. This will avoid you from having a more than thousand files and will make your repo big :+1: